src/java/binary_search/34.find-first-and-last-position-of-element-in-sorted-array.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * @lc app=leetcode id=34 lang=java
+ *
+ * [34] Find First and Last Position of Element in Sorted Array
+ */
+
+// @lc code=start
+class Solution {
+    public int[] searchRange(int[] nums, int target) {
+        int [] ret = {-1, -1};
+        int leftIndex = getLeftIndex( nums, target);
```

Comment:
the "getLeftIndex" and "getRightIndex" look similar. why two different functions,?
can't we first find the leftIndex and just iterate until we find something else to find the rightIndex?

Reply:
Do we even need another function. why not put the while loop in the same function?